x-pack/plugins/security_solution/public/timelines/components/timeline/tabs_content/index.tsx

```diff
@@ -132,6 +134,7 @@ const ActiveTimelineTab = memo<ActiveTimelineTabProps>(
     setConversationId,
     showTimeline,
   }) => {
+    const isEsqlSettingEnabled = useKibana().services.configSettings.ESQLEnabled;
```

Comment:
@logeekal, based on the note in https://github.com/elastic/kibana/pull/173640, I think we need to remove all occurrences of `isEsqlSettingEnabled`.

Reply:
@spong, Conditional render should same as [here](https://github.com/elastic/kibana/pull/174029/files#diff-c9eb10e636b9cedb21368945df018d08cb60b87ccf469be22cbab0abd7ae5ed3R409) without `isEsqlSettingEnabled`. Does that make sense?

Reply:
Yeah, thank you!

Here is the updated conditional without `isEsqlSettingEnabled`:

https://github.com/elastic/kibana/pull/174112/files#diff-c9eb10e636b9cedb21368945df018d08cb60b87ccf469be22cbab0abd7ae5ed3R401